social_website/management/commands/sync_website.py

```diff
@@ -78,8 +78,8 @@ def only_offline(self):
 
     def only_online(self):
         refresh_online_stats()
-        refresh_collection_partner_stats()
-        setup_elastic_search()
+        # refresh_collection_partner_stats()
+        # setup_elastic_search()
```

Comment:
I guess, Lokesh suggested to uncomment these two lines please check this

Reply:
Why these changes in your commits?

Reply:
What was your base branch from which you took branch out, Please confirm

Reply:
sir i was working on master branch then i switched to vl_maaz